### PR TITLE
Fix locators in NavigateToFile page object

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
Change locators in NavigateToFile page object which checks popup of the **Navigate To File** feature.

### CODE REVIEW

**Review by tolusha — CHANGES_REQUESTED** [review]
Add debug id to the all items and use them instead of xpaths

**Comment by SkorikSergey** [comment]
All changed locators are used for searching dynamically created web elements .

**Comment by tolusha** [comment]
This is a suggestion panel. You have to investigate corresponding source code to make assumption.

https://github.com/eclipse/che/blob/3cdf3b6eb713ba8a02fec8b3d148c4ef4363c421/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/navigation/NavigateToFileViewImpl.ui.xml#L74

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/NavigateToFile.java [hunk 1]
```diff
@@ -45,21 +45,21 @@ public NavigateToFile(
 
   private interface Locators {
     String NAVI_TO_FILE_FORM = "gwt-debug-file-navigateToFile-mainPanel";
-    String FILE_NAME_FIELD = "//div[text()='Enter file name:']/following::input[1]";
-    String FILE_NAME_POPUP = "//div[text()='Enter file name:']/following::table[last()]";
+    String FILE_NAME_FIELD = "gwt-debug-navigateToFile-fileName";
```
Comment (tolusha) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/NavigateToFile.java]: The name of the constant should reflect its value.

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/NavigateToFile.java [hunk 2]
```diff
@@ -45,21 +45,21 @@ public NavigateToFile(
...
+    String FILE_NAME_POPUP = "gwt-debug-navigateToFile-suggestionPanel";
```
Comment (tolusha) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/NavigateToFile.java]: The name of the constant should reflect its value.

**Review by tolusha — APPROVED** [review]

**Pull request merged**